hbs-helpers.js

```diff
@@ -88,6 +91,17 @@ function breachMath(lValue, operator = null, rValue = null) {
   return returnValue;
 }
 
+function checkImageSrc(imageName) {
+  try {
+    fs.statSync(`public/img/logos/${imageName}.png`);
```

Comment:
Not all logos are PNG, I think only the occasional SVGs that you manually convert to PNG (for email reasons) are PNG. Not sure if it matters or not, but if we're going to hardcode PNG in places, we may want to consider just dumping the JPG/SVG logos and just using PNG everywhere.

Actually, this may be moot since I think this helper may only be used in the email templates... 🤷‍♂️

Reply:
Also, doing a filesystem statSync is probably pretty heavy if we're calling it for each logo in each email. Not sure if it makes sense to somehow just grab all *.png logos on the filesystem once during server setup and then just check our local object to see if the file exists. Like, if we stored all the logos in a `Map()`, I think we could just use ```EmailLogos.has(`${imageName}.png`)``` or something.